pkg/metrics/metrics.go

```diff
@@ -1,30 +1,49 @@
 package metrics
 
 import (
+	"net/http"
+
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-var (
+type serverMetrics struct {
 	/****Metrics related to Ark server backup***********/
-	lastBackupStatus = prometheus.NewGauge(
-		prometheus.GaugeOpts{
-			Namespace: "heptio-ark", // TODO: don't hard-code this value
-			Name:      "last_backup_status",
-			Help:      "Status of the most recent backup. 1 if it was successful, 0 otherwise",
-		},
-	)
-)
+	lastBackupStatus prometheus.Gauge
+}
+
+var metrics *serverMetrics
```

Comment:
Sorry, 1 more change here, please 😄. I'd prefer not to have any package level variables.